app/controllers/users_controller.rb

```diff
@@ -14,23 +14,23 @@ def show
   def create
     @user = User.new(user_params)
     if @user.save
-      # success
       flash[:success] = 'User Registered on System!'
-      redirect_to user_url(@user)
+      redirect_to @user
```

Comment:
According to [that](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_create.png), you have to redirect to `new_user_path`